server/storeProfile.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+require_once 'config.php';
+
+storeProfile(_$POST);
+
+function storeProfile($data) {
+  GLOBAL $DB_CONN;
+  $response = array();
+  $response['success'] = false;
+
+  $DB_CONN -> beginTransaction();
+
+  try {
+    $clientInsert = "INSERT INTO vita.user
+    (
+      firstName,
+      lastName,
+      phone,
+      email
+    )
+    VALUES
+    (
+      ?,
+      ?,
+      ?,
+      ?
+    );";
+    $userparams = array(
+      $data['firstName'],
+      $data['lastName'],
+      $data['email'],
+      $data['phone']
+    );
+    $stmt = $DB_CONN -> prepare($userInsert)
+    $stmt -> execute($userParams);
+
+    $userabilityID = $DB_CONN -> lastInsterId();
+
+    $abilityInsert = "INSERT INTO vita.userability
+    (
+    languageSkills,
+    taxSkills,
+    taxSkillsType
```

Comment:
Once again, there's no columns in UserAbility called any of these things. Please consult the ERD https://github.com/UNL-CSE-AMBASSADORS/VITA/pull/96. And also, inserting into this table is going to be significantly more complicated than your current setup. We will discuss after standup.